firefox-ios/Client/Frontend/Browser/FirefoxTabContentBlocker.swift

```diff
@@ -113,11 +114,28 @@ class FirefoxTabContentBlocker: TabContentBlocker, TabContentScript {
         )
     }
 
+    private func dispatchSafeListStatusAction(for blockerStatus: BlockerStatus?) {
+        guard featureFlags.isFeatureEnabled(.toolbarRefactor, checking: .buildOnly) else { return }
+
+        let safeListedURLImageName = (
```

Comment:
Why is the if part in brackets here? Looks kinda weird to me.

Reply:
The line was too long so I used the automatic indentation `control + m` and generated it like that. I'll find something nicer though.